test/activities/content/ContentLTILinkEntity.js

```diff
@@ -52,17 +51,6 @@ describe('ContentLTILinkEntity', () => {
 		it('Equality should return false when title is different', () => {
 			const ltiLinkData = {
 				title: 'New Title',
-				descriptionRichText: '<p>description text</p>',
-				url: 'https://phoenix-is-the-best.com',
-				isExternalResource: true
-			};
-			expect(contentLTILinkEntity.equals(ltiLinkData)).to.equal(false);
-		});
-
-		it('Equality should return false when description is different', () => {
```

Comment:
Since both LTI and Weblinks don't have the descriptions in their `equals()` method, these tests can be removed.